src/winter_onboarding_2021/fleet_management_service/views/layout.clj

```diff
@@ -1,13 +1,24 @@
 (ns winter-onboarding-2021.fleet-management-service.views.layout
   (:require [hiccup.page :refer [html5 include-css include-js]]))
 
-(defn application [request title & content]
+(defn application [request title content]
   (html5 [:head
           [:title title]
           [:link {:rel "icon" :href "/public/favicon.ico" :type "image/x-icon"}]
           (include-css "/public/css/bootstrap.min.css")
           (include-js "/public/js/jquery-3.6.0.min.js")
           (include-js "/public/js/bootstrap.bundle.min.js")
+          [:header.d-flex.flex-wrap.justify-content-center.py-3.mb-4.border-bottom
+           [:a.d-flex.align-items-center.mb-3.mb-md-0.me-md-auto.text-dark.text-decoration-none
```

Comment:
Can u format this?

Reply:
On it.

Reply:
@yogirajh007 instead of using the `.` syntax to assign classes, you can do something like `:class "class1 class2"` if you have multiple classes you can let them in a vector and use `clojure.string/join`